SUMMARY.md

```diff
@@ -57,7 +58,10 @@
     * [manifest file](reference/interbit-cli/manifest.adoc)
     * [start](reference/interbit-cli/start.md)
   * [interbit-covenant-tools](reference/interbit-covenant-tools/README.md)
+    * [validate()](reference/interbit-covenant-tools/validate.adoc)
```

Comment:
These function names are not clear when they do not contain the additional scope of the exported object tree. Is there some way we could include the property path from module exports in the documentation for clarity?

Reply:
I may be reading the source incorrectly, but it looks to me like the top-level exports for `interbit-covenant-tools` includes `validate()`. So the documentation hierarchy matches exactly. Are you looking for some sort of breadcrumb, so that when you're viewing `validate()` you can see that it's a top-level function?

As an aside, the `validate()` function's source comments are lacking an `@returns`, and may need review for accuracy.

Reply:
You are correct, I had thought this was one of `validateManifest` or `validateConfig` but it is not. I forgot that export was in `interbit-covenant-tools`. Thanks!